documentation/manual/src/main/asciidoc/modules/infinispan.asciidoc

```diff
@@ -143,21 +143,21 @@ You configure Hibernate OGM and Infinispan in two steps basically:
 * If you need to run JPQL or HQL queries, add Hibernate Search on the classpath
   (<<ogm-query-using-hibernate-search>>)
 
-Note that, except when using [acronym]`JNDI`, Hibernate OGM will bootstrap and Infinispan Embedded node
-in your same JVM, and terminate it on shutdown of the Hibernate instance.
+Note that, except when using [acronym]`JNDI`, Hibernate OGM will bootstrap Infinispan Embedded 
+in the same JVM and terminate it on shutdown of the Hibernate OGM factory.
```

Comment:
What do you think to remove factory, saying just `Hibernate OGM`?

Reply:
I wanted to be clearer. What does it mean to close Hibernate OGM?

Reply:
you're right, it would mean nothing